legal-api/src/legal_api/models/party.py

```diff
@@ -107,6 +107,16 @@ def valid_party_type_data(self):
                 return False
         return True
 
+    @classmethod
+    def find_by_name(cls, first_name: str, last_name: str, organization_name: str):
```

Comment:
If you add
from __future__ import annotations
then you should be able to type the return:
def find_by_name(cls, first_name: str, last_name: str, organization_name: str) -> Party:
Since we're targeting Py3.7+ we should start adding this.